pysb/bng.py

```diff
@@ -395,14 +459,15 @@ def execute(self, reload_netfile=False):
                                  cwd=self.base_directory,
                                  stdout=subprocess.PIPE,
                                  stderr=subprocess.PIPE)
-            if self.verbose:
+            if self.verbose <= logging.DEBUG:
```

Comment:
Performance was my original rationale for this, but there'll only be a lot of lines when BNG's verbose options are set, so this option seems redundant. Will remove.